cfy_manager/components/mgmtworker/mgmtworker.py

```diff
@@ -54,21 +54,31 @@ class MgmtWorker(BaseComponent):
     def __init__(self, skip_installation):
         super(MgmtWorker, self).__init__(skip_installation)
 
-    def _install(self):
-        source_url = config[MGMTWORKER][SOURCES]['mgmtworker_source_url']
-        yum_install(source_url)
+    def _add_snapshot_restore_sudo_commands(self):
+        sudoers.allow_user_to_sudo_command(
+            '/opt/nodejs/bin/npm',
+            description='Allow web UI DB migrations during snapshot restore.',
+            allow_as='stage_user',
+        )
 
-        premium_source_url = config[PREMIUM][SOURCES]['premium_source_url']
-        try:
-            get_local_source_path(premium_source_url)
-        except FileError:
-            logger.info(
-                'premium package not found in manager resources package')
-            logger.notice('premium will not be installed.')
-        else:
-            logger.notice('Installing Cloudify Premium...')
-            cluster = Cluster(skip_installation=False)
-            cluster.install()
+        scripts = {
```

Comment:
tbh a list of pairs would work just as well

Reply:
Changed.